projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -45,11 +45,13 @@ <h2 [innerHtml]="product.nameHtml"></h2>
       </div>
       <div class="col-12 col-md-4">
         <ng-template [cxOutlet]="ProductListOutlets.ITEM_ACTIONS">
+          <!--
```

Comment:
Is fixing this already covered in an another ticket?

Reply:
in ticket #13050